tools/gen-manifests-diff

```diff
@@ -61,9 +61,21 @@ def manifests_diff(tmp_path, rev):
 
 
 def main():
-    rev = "main"
-    if len(sys.argv) > 1:
-        rev = sys.argv[1]
+    match len(sys.argv):
```

Comment:
Note that this now requires Python >= 3.10.

Reply:
Fair point, it seems centos9 has 3.9 - if its a problem I can change it to if/elif/else

Reply:
Since it's a developer tool, I don't think this is an issue.
FTR, [`test/scripts/boot-image`](https://github.com/osbuild/images/blob/a52fe88357ed7beb1d03612949f8206d21280639/test/scripts/boot-image#L172) which runs in our GitLab tests, also uses `match`.